### DISCUSSION TITLE
DLKcat Docker reference format error

### DISCUSSION THREAD

**Discussion by wshao1** [SysBioChalmers/GECKO#350] [Q&A] [answered]
I am trying to create a full ecModel based on Human-GEM 1.16.0, using a modified form of `tutorials/full_ecModel/protocol.m`. Step 7 (DLKcat prediction through machine learning) breaks at the `runDLKcat()` command, giving the following traceback:
![image](https://github.com/SysBioChalmers/GECKO/assets/50877702/20283c47-de72-491d-9ac9-67e45eb8c372)

I've inspected `runDLKcat()`, specifically the `docker run` command at line 38, but it's not clear to me why the reference format is incorrect. Perhaps the filepath is specific to whoever created the tutorial, and we need to change it for our own machine?

**Comment by edkerk** [comment, marked as answer]
Seems more appropriate to address this as an Issue: #351.
